app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -62,25 +61,29 @@ class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
         }
 
 
-    fun onRequestPermissionsResult(
+    internal fun onRequestPermissionsResult(
         requestCode: Int,
         permissions: Array<String>,
         grantResults: IntArray
     ) {
         var locationPermissionGranted = false
         val requestAccess = null
-            if (ContextCompat.checkSelfPermission(this.applicationContext,
-                            Manifest.permission.ACCESS_FINE_LOCATION)
-                    == PackageManager.PERMISSION_GRANTED) {
-                var locationPermissionGranted = true
-            } else {
-
-                //ActivityCompat.requestPermissions(this, arrayOf(Manifest.permission.ACCESS_FINE_LOCATION),
-                        //PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION)
-            }
+        if (ContextCompat.checkSelfPermission(
+                this.applicationContext,
+                Manifest.permission.ACCESS_FINE_LOCATION
+            )
+            == PackageManager.PERMISSION_GRANTED
+        ) {
+            var locationPermissionGranted = true
+        } else {
+
+            //ActivityCompat.requestPermissions(this, arrayOf(Manifest.permission.ACCESS_FINE_LOCATION),
+            //PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION)
+        }
+    }
 
 
-    internal fun onRequestPermissionsResult(requestCode: Int,
+   /* internal fun onRequestPermissionsResult(requestCode: Int,
```

Comment:
There were 2 onRequestPermissionsResult functions which was causing an error, so I commented the one out

Reply:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.